lib/styles/settings/colors/_tokens.scss

```diff
@@ -0,0 +1,304 @@
+$primary-background-strong-hovered: --theme-600;
+$primary-background-strong-focused: --theme-600;
+$primary-background-strong-disabled: --theme-200;
+$primary-background-medium: --theme-100;
+$primary-background: --theme-50;
+$primary-background-hovered: --theme-100;
+$primary-background-focused: --theme-100;
+$primary-background-disabled: --theme-50;
+$primary-background-weak: --raw-white;
+$primary-background-weak-hovered: --theme-100;
+$primary-background-weak-focused: --theme-100;
+$primary-background-weak-pressed: --theme-200;
+$primary-background-ghost: rgba(var(--theme-500), 0);
+$primary-background-ghost-hovered: rgba(var(--theme-500), 0.12);
+$primary-background-ghost-focused: rgba(var(--theme-500), 0.12);
+$primary-background-ghost-pressed: rgba(var(--theme-500), 0.18);
+$primary-background-ghost-disabled: rgba(var(--theme-500), 0);
+$primary-border: --theme-500;
+$primary-border-hovered: --theme-700;
+$primary-border-focused: --theme-700;
+$primary-border-disabled: --theme-200;
+$primary-ripple: rgba(var(--theme-500), 0.12);
+$neutral-text-heavy: --raw-gray-900;
+$neutral-text-heavy-disabled: --raw-gray-400;
+$neutral-text-strong: --raw-gray-800;
+$neutral-text-strong-disabled: --raw-gray-400;
+$neutral-text: --raw-gray-700;
+$neutral-text-hovered: --raw-gray-900;
+$neutral-text-focused: --raw-gray-900;
+$neutral-text-disabled: --raw-gray-400;
+$neutral-text-weak: --raw-gray-500;
+$neutral-text-weak-hovered: --raw-gray-700;
+$neutral-text-weak-focused: --raw-gray-700;
+$neutral-text-weak-disabled: --raw-gray-400;
+$neutral-icon-strong: --raw-gray-800;
+$neutral-icon-strong-disabled: --raw-gray-300;
+$neutral-icon: --raw-gray-600;
+$neutral-icon-hovered: --raw-gray-800;
+$neutral-icon-focused: --raw-gray-800;
+$neutral-icon-disabled: --raw-gray-300;
+$neutral-icon-weak: --raw-gray-400;
+$neutral-icon-weak-hovered: --raw-gray-600;
+$neutral-icon-weak-focused: --raw-gray-600;
+$neutral-icon-weak-disabled: --raw-gray-300;
+$neutral-data-medium: --raw-gray-600;
+$neutral-data: --raw-gray-500;
+$neutral-data-weak: --raw-gray-300;
+$neutral-data-ghost: --raw-gray-200;
+$neutral-background-heavy: --raw-gray-700;
+$neutral-background-heavy-hovered: --raw-gray-800;
+$neutral-background-heavy-focused: --raw-gray-800;
+$neutral-background-heavy-disabled: --raw-gray-300;
+$neutral-background-strong: --raw-gray-400;
+$neutral-background-strong-hovered: --raw-gray-500;
+$neutral-background-strong-focused: --raw-gray-500;
+$neutral-background-strong-disabled: --raw-gray-300;
+$neutral-background-medium: --raw-gray-200;
+$neutral-background: --raw-gray-50;
+$neutral-background-hovered: --raw-gray-100;
+$neutral-background-focused: --raw-gray-100;
+$neutral-background-disabled: --raw-gray-50;
+$neutral-background-weak: --raw-white;
+$neutral-background-weak-hovered: --raw-gray-100;
+$neutral-background-weak-focused: --raw-gray-100;
+$neutral-background-weak-pressed: --raw-gray-200;
+$neutral-background-ghost: rgba(var(--raw-gray-700), 0);
```

Comment:
IMO, this will break. CSS `rgba()` method accepts `red, green, blue, alpha` parameters, not hex value

Reply:
Yeah, You found the bug, there should be `--raw-gray-700-rgb` and it occures everywhere in this file. I will fix it in importer.